### PR TITLE
driftctl: update formula

### PR DESCRIPTION
**Pull request by kwilczynski**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

This change adds a number of improvements to the formula:

- Change how build flags are assembled and set `CGO_ENABLED` to `0`

- Install completion script for _Bash_, _ZSH_ and _Fish_ shells

### CODE REVIEW

**Comment by kwilczynski** [comment]
Hi @wbeuil and @emahe22,

Do let me know what do you think about the proposed changed, and if there is anything you wish to change, update or remove. I would be delighted to help.

Krzysztof

File: Formula/driftctl.rb [hunk 1]
```diff
@@ -15,13 +15,41 @@ class Driftctl < Formula
...
+    <<~EOS
+      Thank you for installing the driftctl!
+
+      To see help and a list of available commands type:
+        $ driftctl --help
+
+      For more information, examples, and documentations visit:
+        https://driftctl.com/
+    EOS
   end
```
Comment (SMillerDev) [inline Formula/driftctl.rb:22]: This is not homebrew specific so can't be a caveat.

File: Formula/driftctl.rb [hunk 2]
```diff
@@ -15,13 +15,41 @@ class Driftctl < Formula
...
 
   test do
+    assert_predicate bin/"driftctl", :exist?
```
Comment (SMillerDev) [inline Formula/driftctl.rb:52]: This is already covered by calling the binary below.

File: Formula/driftctl.rb (hunk 1 above)
Comment (kwilczynski) [reply Formula/driftctl.rb]: Makes sense. Will drop.

**Review by SMillerDev — APPROVED** [review]
Thanks @kwilczynski  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Comment by kwilczynski** [comment]
Hi @SMillerDev,

> Thanks @kwilczynski ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

I appreciate the kind words. It's my pleasure to help a little.

Thank you and others for maintaining Homebrew!

Krzysztof

**Pull request closed without merging**